quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesDistTest.java

```diff
@@ -41,13 +42,14 @@
 @DistributionTest(reInstall = DistributionTest.ReInstall.NEVER)
 @RawDistOnly(reason = "Containers are immutable")
 @TestMethodOrder(OrderAnnotation.class)
+@LegacyStore
```

Comment:
Just out of curiosity, why do we need to run this test with the legacy store?

Reply:
Because one of the steps are relying on enabling metrics for Hibernate. We can certainly review it but I did not want to do it now.